gef.py

```diff
@@ -1740,6 +1740,10 @@ def is_branch_taken(self, insn):
             taken, reason = val&(1<<flags["carry"]) and not val&(1<<flags["zero"]), "C && Z==O"
         elif mnemo.endswith("ls"):
             taken, reason = not val&(1<<flags["carry"]) or val&(1<<flags["zero"]), "C==O || Z"
+        elif mnemo.endswith("mi"):
+            taken, reason = val&(1<<flags["negative"]), "N"
+        elif mnemo.endswith("pl"):
+            taken, reason = not val&(1<<flags["negative"]), "N==O"
```

Comment:
You mean N==0, not N==O. Plus we have been putting `!Z`, so we should be consistent.